tests.yml

```diff
@@ -42,7 +42,7 @@
 
     - name: Compute architectures to download
       set_fact:
-        download_arches: "{{ taskotron_supported_arches | join(',') }}"
+        download_arches: "{{ taskotron_supported_binary_arches | join(',') }}"
```

Comment:
Thanks. Actually, I messed up this a little when converting from `koji download-build` to `download_rpms.py`, so I'll fix it in a following commit.